src/assets/css/oag.css

```diff
@@ -0,0 +1,85 @@
+.oag-theme header .brand img, .oag-theme header .brand object {
+	max-height: none;
+}
+
+.oag-theme header .oag-gb {
+	background: #1a303c;
+}
+
+.oag-theme .btn-primary{
+	background-color: #803C2C;
```

Comment:
If this site is going to have many pages, I suggest to include the GCWeb SCSS instead of CSS and leverage the variables instead of updating every element's colors. However, for a demo, this is fine.

Reply:
Using nested CSS with variables and post css